AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -277,6 +277,7 @@ - (void)_initializeInstance
   _contentsScaleForDisplay = ASScreenScale();
   _displaySentinel = [[ASSentinel alloc] init];
   _preferredFrameSize = CGSizeZero;
+  _pendingViewState = [_ASPendingState new];
```

Comment:
Yes, I have realized progressively more how much of a difference this makes. A follow-up is fine, but it would be a nice optimization to use a C accessor.